apps/meteor/server/models/raw/LivechatRooms.ts

```diff
@@ -67,6 +69,11 @@ export class LivechatRoomsRaw extends BaseRaw<IOmnichannelRoom> implements ILive
 			{ key: { callStatus: 1 }, sparse: true }, // used on statistics
 			{ key: { priorityId: 1 }, sparse: true },
 			{ key: { slaId: 1 }, sparse: true },
+			{ key: { t: 1, ts: 1 } },
+			{ key: { source: 1, ts: 1 }, partialFilterExpression: { source: { $exists: true }, t: 'l' } },
+			{ key: { departmentId: 1, ts: 1 }, partialFilterExpression: { departmentId: { $exists: true }, t: 'l' } },
+			{ key: { 'tags.0': 1, 'ts': 1 }, partialFilterExpression: { 'tags.0': { $exists: true }, 't': 'l' } },
+			{ key: { servedBy: 1, ts: 1 }, partialFilterExpression: { servedBy: { $exists: true }, t: 'l' } },
```

Comment:
Couldn't we keep on the `partialFilterExpression` only?

Reply:
These ones have the extra field for compat. Mongo <5 don't support multiple indexes on the same key even with different partialFilterExp, so having only TS here would be ideal but not possible yet. Put them like that to diff from the others